pkg/proxy/ipvs/proxier_test.go

```diff
@@ -227,7 +227,7 @@ func TestNodePort(t *testing.T) {
 				t.Errorf("Failed to get ipvs destinations, err: %v", err)
 			}
 			for _, dest := range destinations {
-				if dest.Address.To4().String() != epIP || dest.Port != uint16(svcPort) {
+				if dest.Address.String() != epIP || dest.Port != uint16(svcPort) {
```

Comment:
I expected to see new IPv6 tests in here?

Reply:
Sounds fair. I added some tests, please take a loook again?